Added Intros for REST API Pages (#10001)
github/docs: content/rest/reference/actions.md

---
title: Actions

> intro: 'intro: 'The Actions API enables the authenticated user and {% data variables.product.prodname_github_apps %} to manage and control the {% data variables.product.prodname_actions %} in an organization or repositories.'

Reviewer: Let's remove the more technical details for an intro.

```suggestion
intro: 'With the Actions API, you can manage and control {% data variables.product.prodname_actions %} for an organization or repository.'
```

---
title: Actions
intro: 'With the Actions API, you can manage and control {% data variables.product.prodname_actions %} for an organization or repository.'

> =======

Reviewer 2: ```suggestion
```
